### DISCUSSION TITLE
《蓓优妮塔3》PTD文本文件分析笔记

### DISCUSSION THREAD

**Discussion by giscus-xzonn** [Xzonn/xzonn.github.io#10] [General]
# 《蓓优妮塔3》PTD文本文件分析笔记

《蓓优妮塔3》PTD文本文件分析笔记 - 来自Xzonn的小站

https://xzonn.top/posts/Bayonetta-3-Ptd-Text-File-Notes.html

**Comment by Xzonn** [comment]
> 评论导出自来必力

*****Gilson Barbosa Junior 评论于 2023-03-16 05:56:20 (UTC+8)*****

Does not work with "PTD" in Bayonetta Origins Cereza and the Lost Demon. Are you going to update the tool?

*****Xzonn 回复于 2023-03-17 00:05:10 (UTC+8)*****

It works.
